cpp/include/cudf/types.h

```diff
@@ -1,6 +1,7 @@
 #pragma once
 
-typedef size_t gdf_size_type;
+// TODO: Update to use fixed width types when CFFI goes away
```

Comment:
Can you explain this todo?

Reply:
It means using types like `int32_t` that are defined in `<cstdint>` (but we can't use them now because that would require an `include` in a CFFI file :( )